### PR TITLE
Refactor transformer types

### PR DESCRIPTION
**Pull request by mfix-stripe**
PTAL @rpaul-stripe 

* Expose `globalAttributes`
* Move `Transformer` type to `types.ts`

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -129,6 +130,7 @@ export {
   nodes,
   tags,
   functions,
+  attributes,
```
Comment (matv-stripe) [inline index.ts:133]: what is the use case for exposing  this?

```
export const globalAttributes: AttributesSchema = {
  class: { type: Class, render: true },
  id: { type: String, render: true },
};
```

**Review by matv-stripe — COMMENTED** [review]
quick question, @mfix-stripe

File: index.ts (hunk 1 above)
Comment (mfix-stripe) [reply index.ts:133]: Just to align it with the other primitive collections: `nodes`, `tags`, `functions`. Do you think we should call it `globalAttributes` instead?

**Comment by mfix-stripe** [comment]
PTAL @rpaul-stripe — WDYT about exporting this config? Should we call it `globalAttributes` or `attributes`?

**Review by rpaul-stripe — APPROVED** [review]

**Pull request merged**